src/dialogs/index.ts

```diff
@@ -0,0 +1,56 @@
+const preload = () => import("./confirmation-dialog");
+
+export interface ConfirmationDialogParams {
+  text?: string;
+  title?: string;
+  confirmText?: string;
+  dismissText?: string;
+  confirm?: () => void;
+  cancel?: () => void;
+  destructive?: boolean;
+}
+
+export const showConfirmationDialog = (
+  dialogParams: ConfirmationDialogParams
+) =>
+  new Promise((resolve) => {
+    const origCancel = dialogParams.cancel;
+    const origConfirm = dialogParams.confirm;
```

Comment:
Let's remove this. We don't use it and we shouldn't

Reply:
done